workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -301,6 +330,11 @@ def run(self, mount_root, tmp_dir):
             Logger.fail('Unhandled error during mount task: {}.'.format(str(e)), task_name=self.task_name)
             traceback.print_exc()
 
+    def _load_mount_options_from_environ(self):
+        for env_name, env_value in os.environ.items():
+            if env_name.startswith(MOUNT_OPTIONS_ENV_PREFIX):
+                yield int(env_name[len(MOUNT_OPTIONS_ENV_PREFIX):]), MountOptions().parse(env_value)
```

Comment:
Currently, only storage ids are supported as keys in `CP_CAP_MOUNT_OPTIONS_*`. Don't we want to support storage paths here as well? Because otherwise storage recreation will break storage mounting similarly to `CP_CAP_LIMIT_MOUNTS`.

Breaks storage recreation:
```
export CP_CAP_MOUNT_OPTIONS_1="..."
```

Supports storage recreation:
```
export CP_CAP_MOUNT_OPTIONS_MY_STORAGE_PATH="..."
```